webapp/components/wpt-flags.js

```diff
@@ -101,8 +108,27 @@ wpt.FlagsClass = (superClass, readOnly, useLocalStorage) => class extends superC
   static get properties() {
     const props = {};
     makeFeatureProperties(props, wpt.ClientSideFeatures, readOnly, useLocalStorage);
+    if (useLocalStorage) {
+      makeFeatureProperties(props, wpt.ClientSideWriteableFeatures, false, useLocalStorage);
+    }
     return props;
   }
+
+  ready() {
+    super.ready();
+    if (useLocalStorage) {
+      for (const feature of wpt.ClientSideWriteableFeatures) {
+        this._createMethodObserver(`valueChangedInLocalStorage(${feature}, '${feature}')`);
+      }
+    }
+  }
+
+  valueChangedInLocalStorage(value, feature) {
+    window.location.reload();
```

Comment:
Note to reviewer: this line ties to [this comment](https://github.com/web-platform-tests/wpt.fyi/pull/1866#issuecomment-609848180). Ideally we want to only refresh the layout of the page but there is no easy way to do, due to the fact that these flags are passed through mixins, instead of data bindings. Any suggestion is welcomed here

Reply:
Option 2 is less messy and easier, especially when there are multiple writeable client-side flags. But after some unsuccessful attempts doing the following in `wpt.results.js`

`   
  ready() {
    super.ready();
    this.addEventListener('storage', function(e) {
      if (e.key === 'multiselectTriageUI') {
        this.multiselectTriageUI = e.newValue;
        this.reloadData();
      }
    });
  }
`
I realized in https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API/Using_the_Web_Storage_API, it says

> This won't work on the same page that is making the changes — it is really a way for other pages on the domain using the storage to sync any changes that are made. Pages on other domains can't access the same storage objects.

and [another example](https://stackoverflow.com/questions/3562929/how-can-i-get-an-event-to-fire-every-time-localstorage-is-updated-in-safari-5). Guess I will go with option one for now until we